readme-cn.md

```diff
@@ -0,0 +1,1799 @@
+    return { statusCode }
+  }
+
+  render() {
+    return (
+      <p>
+        {this.props.statusCode
+          ? `An error ${this.props.statusCode} occurred on server`
+          : 'An error occurred on client'}
+      </p>
+    )
+  }
+}
+```
+
+<a id="reusing-the-built-in-error-page" style="display: none"></a>
+### 渲染内置错误页面
+
+如果你想渲染内置错误页面，你可以使用`next/error`：
+
+```jsx
+import React from 'react'
+import Error from 'next/error'
+import fetch from 'isomorphic-unfetch'
+
+export default class Page extends React.Component {
+  static async getInitialProps() {
+    const res = await fetch('https://api.github.com/repos/zeit/next.js')
+    const statusCode = res.statusCode > 200 ? res.statusCode : false
+    const json = await res.json()
+
+    return { statusCode, stars: json.stargazers_count }
+  }
+
+  render() {
+    if (this.props.statusCode) {
+      return <Error statusCode={this.props.statusCode} />
+    }
+
+    return (
+      <div>
+        Next stars: {this.props.stars}
+      </div>
+    )
+  }
+}
+```
+
+> 如果你自定义了个错误页面，你可以引入自己的错误页面来代替`next/error`
+
+<a id="custom-configuration" style="display: none"></a>
+### 自定义配置
+
+如果你想自定义Next.js的高级配置，可以在根目录下新建`next.config.js`文件（与`pages/` 和 `package.json`一起）
+
+注意：`next.config.js`是一个Node.js模块，不是一个JSON文件，可以用于Next启动服务已经构建阶段，但是不作用于浏览器端。
+
+```js
+// next.config.js
+module.exports = {
+  /* config options here */
+}
+```
+
+或使用一个函数：
+
+```js
+module.exports = (phase, {defaultConfig}) => {
+  //
+  // https://github.com/zeit/
+  return {
+    /* config options here */
+  }
+}
+```
+
+`phase`是配置文件被加载时的当前内容。你可看到所有的phases常量：[constants](./lib/constants.js)
+这些常量可以通过`next/constants`引入：
+
+```js
+const {PHASE_DEVELOPMENT_SERVER} = require('next/constants')
+module.exports = (phase, {defaultConfig}) => {
+  if(phase === PHASE_DEVELOPMENT_SERVER) {
+    return {
+      /* development only config options here */
+    }
+  }
+
+  return {
+    /* config options for all phases except development here */
+  }
+}
+```
+
+<a id="setting-a-custom-build-directory" style="display: none"></a>
+#### 设置自定义构建目录
+
+你可以自定义一个构建目录，如新建`build`文件夹来代替`.next` 文件夹成为构建目录。如果没有配置构建目录，构建时将会自动新建`.next`文件夹
+
+```js
+// next.config.js
+module.exports = {
+  distDir: 'build'
+}
+```
+
+<a id="disabling-etag-generation" style="display: none"></a>
+#### 禁止etag生成
+
+你可以禁止etag生成根据你的缓存策略。如果没有配置，Next将会生成etags到每个页面中。
+
+```js
+// next.config.js
+module.exports = {
+  generateEtags: false
+}
+```
+
+<a id="configuring-the-ondemandentries" style="display: none"></a>
+#### 配置onDemandEntries
+
+Next暴露一些选项来给你控制服务器部署以及缓存页面：
+
+```js
+module.exports = {
+  onDemandEntries: {
+    // period (in ms) where the server will keep pages in the buffer
+    maxInactiveAge: 25 * 1000,
+    // number of pages that should be kept simultaneously without being disposed
+    pagesBufferLength: 2,
+  }
+}
+```
+
+这个只是在开发环境才有的功能。如果你在生成环境中想缓存SSR页面，请查看[SSR-caching](https://github.com/zeit/next.js/tree/canary/examples/ssr-caching
```

Comment:
Missing a `)` at the end of the URL?

Reply:
Thank you for your suggestion, I have fixed it.